pkg/ocm/flag.go

```diff
@@ -40,6 +49,19 @@ func AddClusterFlag(cmd *cobra.Command) {
 	cmd.RegisterFlagCompletionFunc("cluster", clusterCompletion)
 }
 
+func AddModeFlag(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(
+		&mode,
+		"mode",
+		"m",
+		"",
+		"How to perform the operation. Valid options are:\n"+
+			"auto: To automatically apply command actions\n"+
+			"manual: To output commands which can be used to manually complete command actions",
```

Comment:
I would change this to:
- auto: Resource changes will be automatic applied using the current AWS account
- manual: Commands necessary to modify AWS resources will be output to be run manually